### DISCUSSION TITLE
Mise à jour du CHANGELOG

### DISCUSSION THREAD

**Discussion by cchabot-besse** [dbarzin/mercator#903] [General]
Bonjour, le CHANGELOG n'a pas été mis à jour suite aux dernières livraisons 2024, que ce soit sur la branche master ou les branches 2024-xxx.
Je cherchais notamment à savoir si la fonctionnalité permettant d’ajouter des attributs personnalisés était disponible (il semble que non).

**Comment by dbarzin** [comment]
Tu as raison, le CHANGELOG n’a pas été mis à jour comme il aurait dû l’être pour les dernières versions de 2024, que ce soit sur master ou les branches 2024-xxx. Désolé pour l’oubli.

Pour la fonctionnalité d’ajout d’attributs personnalisés, elle n’est pas encore dispo. C’est une grosse modif qui demande pas mal de boulot, donc on y réfléchit encore. Si jamais tu veux contribuer ou donner des idées, on est preneurs ! Maintenant, ce sera certainement dispo premier semestre de l'année prochaine.
